atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionTest.java

```diff
@@ -490,11 +501,8 @@ public void testRangesTransaction() {
 
         RangeRequest allRange = RangeRequest.builder().batchHint(3).build();
         t = startTransaction();
-        final Iterable<BatchingVisitable<RowResult<byte[]>>> ranges = t.getRanges(TEST_TABLE, Iterables.limit(Iterables.cycle(allRange), 1000));
-        for (BatchingVisitable<RowResult<byte[]>> batchingVisitable : ranges) {
-            final List<RowResult<byte[]>> list = BatchingVisitables.copyToList(batchingVisitable);
-            assertEquals(1, list.size());
-        }
+
+        verifyAllGetRangesImplsRangeSizes(t, Iterables.limit(Iterables.cycle(allRange), 1000), 1);
```

Comment:
You do `Iterables.limit(Iterables.cycle(foo))` each time. Does it make sense to push that down to the helper method?